.appveyor.yml

```diff
@@ -0,0 +1,27 @@
+# Builds are whole numbered
+version: '{build}'
+
+init:
+  - git config --global core.autocrlf input
```

Comment:
It's not mandatory and the builds I've tested without it have been successful.

I kept it here as a safeguard for future actions (like zipping/deploying) so that the unix-style line endings that we have in the code will not be altered when cloning.